setup.py

```diff
@@ -35,14 +35,35 @@
 with open(".dep-versions") as f:
     jax_version = [line[4:].strip() for line in f.readlines() if "jax=" in line][0]
 
+pl_version = environ.get("pl_version", ">=0.32,<=0.34")
 requirements = [
-    "pennylane>=0.32",
+    f"pennylane{pl_version}",
     f"jax=={jax_version}",
     f"jaxlib=={jax_version}",
     "tomlkit;python_version<'3.11'",
     "scipy",
 ]
 
+one_compiler_per_distribution = pl_version == ">=0.32,<=0.34"
```

Comment:
Could you add a TODO as this is temporary and will be removed with the next release?

Reply:
See here: https://github.com/PennyLaneAI/catalyst/pull/493/commits/9702c7ca3a655cfcf951db63e2f9e5bc473e4199